src/Dashboard/Search/KingRichardsKnight/Solution.java

```diff
@@ -0,0 +1,123 @@
+import java.io.*;
+import java.util.*;
+import java.text.*;
+import java.math.*;
+import java.util.regex.*;
+
+public class Solution
+{
```

Comment:
For sake of convention:
- Let's keep the opening of the brackets/braces/parenthesis in the end of the line not a new line.
- Remove unnecessary blank lines within the code.
- Keep just a blank line in the end of each file (known as EOF). (You already made it).

Please apply this in all.